src/math_utils.rs

```diff
@@ -92,7 +101,7 @@ pub fn ec_add(
 /// Assumes the points are given in affine form (x, y) and have different x coordinates.
 pub fn line_slope(point_a: (BigInt, BigInt), point_b: (BigInt, BigInt), prime: &BigInt) -> BigInt {
```

Comment:
An the same for `point_a` and `point_b`.

Reply:
resolved: 
https://github.com/lambdaclass/cleopatra_cairo/pull/333/commits/fdcfbe283adf510aaf7cb1982f6c256f43f8ec64